src/ball.rs

```diff
@@ -8,22 +11,50 @@ const BALL_START_POS: Pos2 = Pos2::new(0.0, 0.0);
 
 // TODO: Move to be read from the configuration file!
 // This is specified in the normalized vector position units per second
-const BALL_START_VELOCITY: Vec2 = Vec2::new(0.25, 0.35);
```

Comment:
would be nice for these to be scalars, then you generate a unit vector in a random direction because it looks like the balls will always move in the same direction before it randomly changes the first time